core/src/gl/shaderProgram.cpp

```diff
@@ -11,6 +11,42 @@
 
 namespace Tangram {
 
+namespace Uniform {
```

Comment:
Looks a lot like a scoped enum to me! If we made this an `enum class : uint16_t` I think we could avoid duplicating this list.